monai/utils/misc.py

```diff
@@ -890,11 +890,11 @@ def is_sqrt(num: Sequence[int] | int) -> bool:
     return ensure_tuple(ret) == num
 
 
-def unsqueeze_right(arr: NdarrayOrTensor, ndim: int) -> NdarrayOrTensor:
```

Comment:
Was there a reason to change this? I felt these might be useful with ndarrays as well as tensors.

Reply:
i was struggling with mypy errors and this was the only solution I could find to fix them. I agree it would be more useful as it was; if you've any suggestions for returning it to how it was and getting the schedulers to work with mypy that would be great

Reply:
We can come back to it later with the refactor I guess.